src/videojs-contrib-hls.js

```diff
@@ -601,7 +601,9 @@ videojs.m3u8 = m3u8;
 videojs.options.hls = videojs.options.hls || {};
 
 if (videojs.registerPlugin) {
-  videojs.registerPlugin('reloadSourceOnError', reloadSourceOnError);
+  if (!videojs.getPlugin('reloadSourceOnError') {
```

Comment:
I would suggest deregistering the plugin if it exists so players would get the "newer" version of the plugin

Reply:
Great suggest